compiler_opt/es/blackbox_optimizers_test.py

```diff
@@ -162,18 +159,23 @@ def testProjectedGradientOptimizer(self):
     The exact solution is (0,1).
     """
 
-    def cost_function(x):
-      return (x[0] + 1)**2 + (x[1] - 1)**2
+    class GenericFunction(bo.QuadraticModel):
+
+      def __init__(self):  # pylint: disable=super-init-not-called
+        pass
+
+      def f(self, x):
+        return (x[0] + 1)**2 + (x[1] - 1)**2
 
-    def cost_gradient(x):
-      return np.array([2 * (x[0] + 1), 2 * (x[1] - 1)])
+      def grad(self, x):
+        return np.array([2 * (x[0] + 1), 2 * (x[1] - 1)])
 
     def projector(x):
       return np.maximum(0, x)
 
-    objective_function = SecondorderBlackboxOptimizersTest.GenericFunction()
-    objective_function.f = cost_function
-    objective_function.grad = cost_gradient
+    objective_function = GenericFunction()
+    # objective_function.f = cost_function
```

Comment:
remove commented code (we don't commit commented code)

Reply:
The commented code has been removed